diff --git a/CMakeLists.txt b/CMakeLists.txt
index c43b18c..dd35eef 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -318,9 +318,7 @@ if (USE_PYTHON)
   if(CMAKE_CXX_FLAGS MATCHES "-Wshadow")
     target_compile_options(gemmi PRIVATE "-Wno-shadow")
   endif()
-  if(WIN32 OR CYGWIN OR NOT ZLIB_FOUND)
-    support_gz(gemmi)
-  endif()
+  support_gz(gemmi)
   if (NOT DEFINED PYTHON_INSTALL_DIR)
     set(PYTHON_INSTALL_DIR ${PYTHON_SITE_PACKAGES})
   endif()
